projects/storefrontapp-e2e-cypress/cypress/integration/regression/anonymous-consents/anonymous-consents-flow.e2e-spec.ts

```diff
@@ -6,14 +6,14 @@ import {
   sessionLogin,
   testAsAnonymousUser,
   testAsLoggedInUser,
-} from '../../helpers/anonymous-consents';
+} from '../../../helpers/anonymous-consents';
 import {
   LANGUAGES,
   LANGUAGE_REQUEST,
   stub,
-} from '../../helpers/site-context-selector';
+} from '../../../helpers/site-context-selector';
 
-context('Anonymous consents', () => {
+context('Anonymous consents - normal flow', () => {
```

Comment:
Let's rename this to something more descriptive. _normal_ is not the same to everyone ;)